packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -121,8 +131,40 @@ export const connectEpic: Epic<AnyAction, AnyAction, any, any> = (action$, state
         }),
       )
 
+      const messages$ = action$.pipe(
+        filter(messageReceived.match),
+        mergeMap((action) => 
+          of(action).pipe(
+            filter((action) => !wsHandler.filter(action.payload.message)),
+            switchMap(async (action) => {
```

Comment:
What's the reasoning for a `switchMap` here? Do we want to drop the current caching action if a new message comes in before it finishes? If we don't, we should use `mergeMap` instead